chemie/events/templates/events/bedpres/detail.html

```diff
@@ -107,8 +107,15 @@ <h5 class="left-align">Plasser</h5>
     </div>
   </div>
 
-  <!-- Button for signup page -->
-  <a class="waves-effect waves-light btn" href="{{ bedpres.get_absolute_registration_url }}">Til påmelding</a>
+  <div class="row">
+      <!-- Button for signup page -->
+      <a class="waves-effect waves-light btn" href="{{ bedpres.get_absolute_registration_url }}">Til påmelding</a>
+      <!-- Button for check in page -->
+      {% if perms.events.change_bedpresregistration %}
+      <a class="waves-effect waves-light btn" href="/events/bedpres/{{ bedpres.pk }}/checkin/">Til innsjekk</a>
```

Comment:
I want to change this href to href="{{ events:checkin_bedpres }}" which is the url namespace, so that it is consistent with changes in urls.py. But the url needs a pk variable, and I don't know how to pass it through the template. Any ideas? @hambro @sindreoy @haakonbu

Reply:
You can write something like
```
href="{% url events:checkin_bedpres pk %}"

Reply:
yes
`href="{% url 'events:checkin_bedpres' bedpres.pk %}` worked